README.md

```diff
@@ -8,3 +8,13 @@ Learn about the Project Haiku (former Project Smart Home) on the Mozilla wiki
 Follow our progress in our Github wiki
 * [https://github.com/mozilla/project_haiku.iot/wiki](https://github.com/mozilla/project_haiku.iot/wiki)
 * Github won't notify you when this wiki is updated, but you can [subscribe via RSS](https://github.com/mozilla/project_haiku.iot/wiki.atom).
+
```

Comment:
Can we move this to status-server/README.md? For now, it makes more sense in that context though we may want pointers from here also eventually.